docker/src/test/java/org/jclouds/docker/compute/SshToCustomPortLiveTest.java

```diff
@@ -221,4 +202,11 @@ private static void consumeStreamSilently(InputStream is) {
    private static String toTag(String repo, String tag) {
       return repo + (tag != null ? ":" + tag : "");
    }
+
+   private void removeImageIfExists(String imageTag) {
```

Comment:
Make this statically available in some base test class so it can be reused instead of repeated.